test/spyOn.js

```diff
@@ -0,0 +1,31 @@
+function spyOn (obj, property) {
```

Comment:
Why not to use `sinon.spy` from [sinon](http://npmjs.com/package/sinon) lib?

Reply:
Didn't want to bring in yet another library. Why not just use Jest?